PurchasesTests/Mocks/MockLegacySKProduct.swift

```diff
@@ -3,8 +3,9 @@
 // Copyright (c) 2020 Purchases. All rights reserved.
 //
 import StoreKit
+@testable import RevenueCat
 
-class MockSKProduct: SKProduct {
+class MockSK1Product: SK1Product {
```

Comment:
The name of this file should be `MockSK1Product.swift`

Reply:
oops, good catch